### PR TITLE
fix(docs) Fixed the rendering issues

### PR DESCRIPTION
**Pull request by Rajakavitha1**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by paulb-seldon** [comment]
A note to revert the overwriting of the changes made right before this PR to Architecture and Architecture/Dataflow pages (see [here](https://github.com/SeldonIO/seldon-core/pull/6013))

**Comment by paulb-seldon** [comment]
Some other notes - I seem to have several pages where blocks aren't working / loading, and where I see this:

Screenshot 2024-10-30 at 11 44 14

The pages are:
* Configuration > Kubernetes
* Kubernetes > Resources > Pipeline block not working
* Kubernetes > Resources > Seldon Config block not working
* Inference Page (several blocks)

Separately, In the Model Zoo example the images do not render correctly


Apart from these, LGTM. Thank you for this change!

File: docs-gb/examples/experiment-versions.md
Comment (lc525) [inline docs-gb/examples/experiment-versions.md:44]: Those are outputs rather than bash (everything that starts with Success)

File: docs-gb/examples/k8s-examples.md
Comment (lc525) [inline docs-gb/examples/k8s-examples.md:59]: Same comment regarding output/bash for "condition met", "deleted", etc. If the formatting is ok then we can leave them as bash

File: docs-gb/getting-started/configuration.md
Comment (lc525) [inline docs-gb/getting-started/configuration.md:32]: Not specifically for this change, but please note you can reference a line-range within a given file, like this:

`https://github.com/SeldonIO/seldon-core/blob/v2/k8s/yaml/components.yaml#L20-L47`

File: docs-gb/getting-started/kubernetes-installation/README.md
Comment (lc525) [inline docs-gb/getting-started/kubernetes-installation/README.md:8]:
```suggestion
## Install Ecosystem Components
```

File: docs-gb/kubernetes/resources/model.md
Comment (lc525) [inline docs-gb/kubernetes/resources/model.md:12]: Please reference the file via a @github-files/github-code-block. As much as possible, we want to change things in a single place, with the files in the repo being the source-of truth.

File: docs-gb/kubernetes/resources/pipeline.md
Comment (lc525) [inline docs-gb/kubernetes/resources/pipeline.md:5]: Please reference the file via a @github-files/github-code-block. As much as possible, we want to change things in a single place, with the files in the repo being the source-of truth.

File: docs-gb/kubernetes/resources/pipeline.md
Comment (lc525) [inline docs-gb/kubernetes/resources/pipeline.md:36]: Please reference the file via a @github-files/github-code-block. As much as possible, we want to change things in a single place, with the files in the repo being the source-of truth.

File: docs-gb/kubernetes/resources/seldonconfig.md
Comment (lc525) [inline docs-gb/kubernetes/resources/seldonconfig.md:80]: Please reference the file via a @github-files/github-code-block. As much as possible, we want to change things in a single place, with the files in the repo being the source-of truth.

File: docs-gb/kubernetes/service-meshes/ambassador.md
Comment (lc525) [inline docs-gb/kubernetes/service-meshes/ambassador.md:393]: This is python notebook code. The underlying command (without the !) is just bash.

**Review by lc525 — CHANGES_REQUESTED** [review]
Thank you for the PR, Rajie! I've added some comments suggesting how we can improve, as well as a discussion to be had with gitbook to understand more about what they suggest wrt file includes.

File: docs-gb/kubernetes/resources/experiment.md
Comment (Rajakavitha1) [reply docs-gb/kubernetes/resources/experiment.md:70]: Hi @lc525  I tried to look for the example files here: https://github.com/SeldonIO/seldon-core/tree/v2/k8s/samples. I am not able to locate the samples/experiments/ab-default-model.yaml, and others.

File: docs-gb/kubernetes/resources/pipeline.md
Comment (Rajakavitha1) [reply docs-gb/kubernetes/resources/pipeline.md:5]: Hi @lc525  I am not able to file the files here: https://github.com/SeldonIO/seldon-core/tree/v2/k8s/samples

File: docs-gb/kubernetes/resources/pipeline.md
Comment (Rajakavitha1) [reply docs-gb/kubernetes/resources/pipeline.md:36]: Hi @lc525 please help me with the location of file. Unable to find here: https://github.com/SeldonIO/seldon-core/tree/v2/k8s/samples

File: docs-gb/examples/experiment-versions.md
Comment (Rajakavitha1) [reply docs-gb/examples/experiment-versions.md]: fixed.

File: docs-gb/examples/k8s-examples.md
Comment (Rajakavitha1) [reply docs-gb/examples/k8s-examples.md]: fixed

File: docs-gb/getting-started/configuration.md
Comment (Rajakavitha1) [reply docs-gb/getting-started/configuration.md:32]: line range not rendering in GitBook.

File: docs-gb/kubernetes/resources/model.md
Comment (Rajakavitha1) [reply docs-gb/kubernetes/resources/model.md:12]: unable to find the file

File: docs-gb/kubernetes/resources/seldonconfig.md
Comment (Rajakavitha1) [reply docs-gb/kubernetes/resources/seldonconfig.md:80]: unable to find the file

File: docs-gb/kubernetes/resources/seldonconfig.md
Comment (Rajakavitha1) [reply docs-gb/kubernetes/resources/seldonconfig.md:16]: okay!

File: docs-gb/kubernetes/service-meshes/ambassador.md
Comment (Rajakavitha1) [reply docs-gb/kubernetes/service-meshes/ambassador.md]: fixed

File: docs-gb/kubernetes/resources/experiment.md
Comment (paulb-seldon) [reply docs-gb/kubernetes/resources/experiment.md:70]: @Rajakavitha1 I believe these yamls are under https://github.com/SeldonIO/seldon-core/tree/v2/samples and then under the right resource (e.g. experiments in this case)

File: docs-gb/kubernetes/resources/model.md
Comment (paulb-seldon) [reply docs-gb/kubernetes/resources/model.md:12]: Same as above I think the files are under v2/samples/{resource} so in this case: 
https://github.com/SeldonIO/seldon-core/blob/v2/samples/models/sklearn-iris-gs.yaml

File: docs-gb/kubernetes/resources/pipeline.md
Comment (paulb-seldon) [reply docs-gb/kubernetes/resources/pipeline.md:5]: Here: https://github.com/SeldonIO/seldon-core/blob/v2/samples/pipelines/tfsimples-join.yaml

File: docs-gb/kubernetes/resources/seldonconfig.md
Comment (paulb-seldon) [reply docs-gb/kubernetes/resources/seldonconfig.md:80]: Here: https://github.com/SeldonIO/seldon-core/blob/v2/operator/config/serverconfigs/mlserver.yaml (see path commented at the top). Perhaps worth considering if all source files need to be within the docs folder?

File: docs-gb/kubernetes/resources/serverconfig.md
Comment (paulb-seldon) [reply docs-gb/kubernetes/resources/serverconfig.md:14]: see above

File: docs-gb/kubernetes/resources/experiment.md
Comment (Rajakavitha1) [reply docs-gb/kubernetes/resources/experiment.md]: Fixed.

File: docs-gb/kubernetes/resources/pipeline.md
Comment (Rajakavitha1) [reply docs-gb/kubernetes/resources/pipeline.md:36]: fixed

File: docs-gb/kubernetes/resources/model.md
Comment (Rajakavitha1) [reply docs-gb/kubernetes/resources/model.md:12]: fixed

File: docs-gb/kubernetes/resources/pipeline.md
Comment (Rajakavitha1) [reply docs-gb/kubernetes/resources/pipeline.md]: fixed

**Review by paulb-seldon — APPROVED** [review]
LGTM - though I'm not sure if all of the files across our docs are now rendered from source files (and not sure how to check). In any case I propose if there are more to port over to reference source files it be done as a follow up PR for now? Wyt @Rajakavitha1 @lc525 ?

File: docs-gb/getting-started/configuration.md
Comment (lc525) [reply docs-gb/getting-started/configuration.md:32]: It appears ok when I add it directly via the gitbook app. See [here](https://app.gitbook.com/o/rOXFWV2k010iw8QdOyaH/s/n8ONqKsfp9k7WHp5OWSM/~/changes/81/getting-started/configuration). Does this then not show up when rendering the actual page? That would be a bug that we should report.

**Review by lc525 — APPROVED** [review]
Lgtm, let's iterate with gitbook regarding line-ranges and marker-based code references.

**Comment by Rajakavitha1** [comment]
Hi @paulb-seldon  please merge this PR. @lc525  I will fix the line rendering vis the GitBook app after this PR is merged is that okay?

File: docs-gb/getting-started/configuration.md
Comment (Rajakavitha1) [reply docs-gb/getting-started/configuration.md:32]: Hi @lc525  I checked the preview and it renders fine. https://docs.seldon.ai/seldon-core-2/getting-started/configuration

**Pull request merged**
